### PR TITLE
Upgrade node10

### PR DESCRIPTION
**Pull request by Crash--**
libxslt is not compatible with node 10 (https://github.com/albanm/node-libxslt/issues/68) so I changed to `xsltproc` so I needed to rewrite some part of `customizeConfig.js`

Lot of `yarn.lock` changes since I had to remove a lot of package before being able to install the right package versions.

Seems that I had to regenerate a few snapshots because of some indents 

The idea behind was to be able to use this webpack plugin https://www.npmjs.com/package/webpack-plugin-ramdisk

### CODE REVIEW

**Comment by ptbrowne** [comment]
> so I changed to xsltproc

Woot, this means that I will not have the native library always rebuilding.

File: src/targets/mobile/scripts/customizeConfigXML.js [hunk 1]
```diff
@@ -64,29 +40,31 @@ const customizeConfigXML = function(context) {
...
+        .replace('$IOS_VERSION_CODE', overrideConfig.fullVersion)
+        .replace('$USER_AGENT_VERSION', 'io.cozy.banks.mobile-' + versionCode)
 
       fs.writeFileSync(configPath, output)
-      console.log(
-        `config.xml successfully transformed using ${transformFilePath}`
-      )
-    })
+    }
+    fs.unlinkSync(tmpPath)
```
Comment (ptbrowne) [inline src/targets/mobile/scripts/customizeConfigXML.js]: The temp file is not unlinked if there is a problem. It is not particularly important here but when there are resources, it is good to have the habit of 

```
initializeResource()
try {
 doStuffThatMightThrow()
} finally {
  destroyResource()
}
```

In python, this is built into the language with the `with` statement. A similar approach could be used in JS.

```
withTempFile(tempFile => {
 // do stuff
})
```

**Comment by ptbrowne** [comment]
:+1: Thanks !

File: src/targets/mobile/scripts/customizeConfigXML.js (hunk 1 above)
Comment (Crash--) [reply src/targets/mobile/scripts/customizeConfigXML.js]: Is it better now?

Comment (ptbrowne) [reply src/targets/mobile/scripts/customizeConfigXML.js]: Yes :) Thanks !

**Pull request merged**